mkdocs.yml

```diff
@@ -67,23 +30,21 @@ nav:
           - components/qa-service/index.md
           - Semantic Search: components/qa-service/semantic-search.md
           - LLM Response: components/qa-service/llm-response.md
-      - Triaging Service:
-          - components/triaging/index.md
```

Comment:
Should we put this back? Although it's not there yet. cc: @Tanmay-97

Reply:
Happy to drop this till we have it. @suzinyou